app/api-v1/routes/members/{aliasOrAddress}.js

```diff
@@ -16,9 +16,11 @@ module.exports = function (apiService) {
 
       if (addrRegex.test(aliasOrAddress)) {
         result = await apiService.getMembersByAddress(aliasOrAddress)
-      }
-      if (aliasRegex.test(aliasOrAddress)) {
+      } else if (aliasRegex.test(aliasOrAddress)) {
         result = await apiService.getMembersByAlias(aliasOrAddress)
+      } else {
+        res.status(400)
+        return
```

Comment:
Is there a test case you can produce that hit's this? Perhaps a string longer than 255 characters?

Reply:
I have added a test locally that does not fail with a `>255` length string.

Reply:
So we need to make that fail somehow. Perhaps the `Alias` regex should be `?!^[1-9A-HJ-NP-Za-km-z]{48}$)^.{0,255}$` instead? This seems to work:

```js
/(?!^[1-9A-HJ-NP-Za-km-z]{48}$)^.{0,255}$/.test(Array(256).fill('a').join('')) // false
/(?!^[1-9A-HJ-NP-Za-km-z]{48}$)^.{0,255}$/.test(Array(256).fill('a').join('')) // true
```